packages/compartment-mapper/src/import-hook.js

```diff
@@ -95,6 +93,17 @@ const nodejsConventionSearchSuffixes = [
   '/index.node',
 ];
 
+/**
+ * Computes the relative path to a module from its compartment location (including a leading `./`)
+ *
+ * @param {string} moduleSpecifierLocation Absolute path (not URL) to a module specifier
+ * @param {string} location Compartment location
+ * @returns {string} Redirect static module interface; the `specifier` prop of this value _must_ be a relative path with leading `./`
+ */
+const relativeSpecifier = (moduleSpecifierLocation, location) => {
+  return `./${moduleSpecifierLocation.replace(location, '')}`;
+};
```

Comment:
I still cringe at the possibility of returning `./${moduleSpecifierLocation}`

Here's a version that should be equaly performant and more strict.

```suggestion
const relativeSpecifier = (moduleSpecifierLocation, location) => {
  assert(moduleSpecifierLocation.startsWith(location), 
    `Module specifier location "${moduleSpecifierLocation}" does not start with compartment location "${location}"`);
  return `./${moduleSpecifierLocation.substring(location.length)}`;
};
```